roles/config/cluster/base/templates/configs/tls-7.1.0.j2

```diff
@@ -96,6 +96,10 @@ KNOX:
     ssl_enabled: true
     ssl_server_keystore_location: {{ tls_keystore_path_generic }}
     ssl_server_keystore_password: {{ tls_keystore_password }}
+KUDU:
```

Comment:
nor does this

Reply:
This one is - I had to pull the Kudu truststore settings out of tls.j2 and into tls-7.1.0.j2